### ISSUE TITLE
Accessibility: Interface: Missing text size scaling for numbered items in player inventory.

### ISSUE THREAD

**Issue by jrotunda** [Meridian59/Meridian59#1197]
The in-game options menu allows for adjustment to the scaling of most text that appears  (text, letter, number size) throughout the interface.

Currently, there is no option to scale the size of the text that appears in the player inventory in the far right of a 'stacked item'. 

Meridian's Stacked items are: 

1 - money/shillings
2 - all food
3 - all ammo (bow arrows, crossbow bolts) 
4 - all reagents

With the upgrades to resolution and technology available to players with larger, higher resolution monitors, this missing scaling elements has been asked about. 

Despite its minor use, it's expected there be some way to adjust the font to some degree here so that a user can view the numbers better of their crucial supplies (both PvE and PvP players on 101 have asked about this recently).

Attached is a screenshot of where it's expected this option be.  Discussing with both a casual player and a pvp vet (Bjorn) 'Player Inventory' or simply 'Inventory' were suggested and would be understandable, but up to the coder.

<img width="1158" height="530" alt="Image" src="https://github.com/user-attachments/assets/370cdc22-34e6-4442-945f-2d42c2875e99" />
<img width="138" height="55" alt="Image" src="https://github.com/user-attachments/assets/782cef23-7f81-4169-bea7-89b1e7a85238" />
